### PR TITLE
consolidate coreDNS version to ease dependency bump

### PR DESCRIPTION
**Pull request by yastij**
Signed-off-by: Yassine TIJANI <ytijani@vmware.com>

**What type of PR is this?**

/kind cleanup
/priority backlog

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**: following #79366 we need to consolidate our version declaration, this would ensure that we bump an external dependency without forgetting something

**Special notes for your reviewer**:

/assign @liggitt @dims 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/compute_test.go [hunk 1]
```diff
@@ -34,6 +34,10 @@ import (
 	etcdutil "k8s.io/kubernetes/cmd/kubeadm/app/util/etcd"
 )
 
+const (
+	coreDNSVersion = "1.3.1"
```
Comment (rajansandeep) [inline cmd/kubeadm/app/phases/upgrade/compute_test.go]: Can we just use `constants.CoreDNSVersion` instead of declaring a new constant here?

Comment (justaugustus) [reply cmd/kubeadm/app/phases/upgrade/compute_test.go]: +1
Also, TIL about `constants.Foo`

Comment (yastij) [reply cmd/kubeadm/app/phases/upgrade/compute_test.go]: Done

**Review by SataQiu — APPROVED** [review]
Thanks @yastij 
/LGTM

**Review by neolit123 — COMMENTED** [review]
thanks
/approve
@yagonobre you had ideas about having coredns with a version map i believe?

**Comment by yagonobre** [comment]
@neolit123, yes I'm planning to change it to a map, to use on some tests.

**Pull request merged**